cs_tools/api/models/metadata.py

```diff
@@ -368,3 +370,52 @@ def details(
                 }
             )
         return r
+
+    @validate_arguments
+    def tml_export(
+        self,
+        export_ids: List[GUID],
+        format_type: TMLType = TMLType.yaml,
+        export_associated: bool = False,
+    ) -> httpx.Response:
+        """
+        Details of one or more metadata objects in the repository.
+        """
+        r = self.rest_api.request(
+                'POST',
+                'metadata/tml/export',
+                privacy='public',
+                data={
+                    'export_ids': stringified_array(export_ids),
+                    'format_type': format_type.value,
+                    'export_associated': export_associated
+                }
+            )
+        return r
+
+    @validate_arguments
+    def tml_import(
+            self,
+            import_objects: List[str],
+            import_policy: TMLImportPolicy = TMLImportPolicy.validate_only,
+            force_create: bool = False,
+    ) -> httpx.Response:
+        """
+        Details of one or more metadata objects in the repository.
+        :param import_objects: List of TML objects as YAML format
+        :param import_policy: The import policy to use (q.v.)
+        :param force_create: If true, force creation of new objects.
+        """
+
+        r = self.rest_api.request(
+            'POST',
+            'metadata/tml/import',
+            headers={'Accept': 'text/plain'},
+            privacy='public',
+            data={
+                'import_objects': json.dumps(import_objects),
+                'import_policy': import_policy.value,
+                'force_create': str(force_create).lower()
+            }
```

Comment:
you might be able to use `self.rest_api.request(..., json={ ... })` instead of data keyword argument. this should remove the need to force a json string on `import_objects` and `force_create`.

Reply:
It works.  I'll leave as is.